runtime/compiler/CMakeLists.txt

```diff
@@ -291,8 +290,16 @@ else()
 	message(SEND_ERROR "unsupported platform")
 endif()
 
-omr_stringify(J9_CFLAGS_STR ${J9_CFLAGS} ${J9_SHAREDFLAGS})
-omr_stringify(J9_CXXFLAGS_STR ${J9_CXXFLAGS} ${J9_SHAREDFLAGS})
+if(J9VM_OPT_JITSERVER)
+	list(APPEND J9_CXXFLAGS -std=c++11)
+endif()
```

Comment:
> This change belongs in a separate pull request; it is unrelated to the purpose here.

Good point. I'll remove it from this PR.

> Further, I'd argue that if any of the JIT code needs C++11, then it should always be compiled that way.

It looks like all the compilers currently used to build JITServer don't distinguish between `c++0x` and `c++11`, so this change is not necessary. At least for GCC the `c++0x` option is deprecated, but equivalent to `c++11`. I'm not sure about other compilers. I added this change only because I noticed an inconsistency when switching to the CMake build from the UMA build.